db/index.js

```diff
@@ -1,14 +1,48 @@
-import mysql from 'mysql';
-import mysqlConfig from './config';
+const Sequelize = require('sequelize');
+const mysqlConfig = require('./config');
+const { deals } = require('./models');
+const { packages } = require('./models');
 
-const connection = mysql.createConnection(mysqlConfig);
+const sequelize = new Sequelize(mysqlConfig);
```

Comment:
I would suggest using a different variable name to differentiate it from the ORM. Maybe `db` or `database`?

Reply:
It is all throughout their documentation and I vaguely remember reading something about using sequelize specifically. Of course I cannot find that now.